tests/resources/Nimbus-Util.robot

```diff
@@ -498,7 +498,6 @@ Power Off Host
 Create Simple VC Cluster With Static IP
     [Arguments]  ${name}=vic-simple-vc-static-ip
     [Timeout]    110 minutes
-    Set Suite Variable  ${NIMBUS_LOCATION}  NIMBUS_LOCATION=wdc
```

Comment:
It seems like this is set in other places. Should those be updated as well for consistency?

Have we looked at the history of these files to determine if this was introduced for any specific reason?

Reply:
It doesn't look like this PR was updated to cover those other places. Could you file a separate issue for that?